src/start/RealmScreen.js

```diff
@@ -85,10 +101,16 @@ class RealmScreen extends PureComponent<Props, State> {
         {error && <ErrorMsg error={error} />}
         <ZulipButton
           style={styles.smallMarginTop}
-          text="Enter"
+          text="+ Add"
```

Comment:
`Enter` will be more appropriate here.

Reply:
@jainkuniya OK, I changed it back to `Enter`. Thanks!